src/net.cpp

```diff
@@ -944,11 +946,11 @@ void ThreadSocketHandler2(void* parg)
                 if (nErr != WSAEWOULDBLOCK)
                     LogPrintf("socket error accept INVALID_SOCKET: %d", nErr);
             }
-            else if (nInbound >= gArgs.GetArg("-maxconnections", 125) - MAX_OUTBOUND_CONNECTIONS)
+            else if (nInbound >= max_connections - MAX_OUTBOUND_CONNECTIONS)
             {
                 LogPrint(BCLog::LogFlags::NET,
                          "Surpassed max inbound connections maxconnections:%" PRId64 " minus max_outbound:%i",
-                         gArgs.GetArg("-maxconnections", 125),
+                         max_connections,
                          MAX_OUTBOUND_CONNECTIONS);
```

Comment:
MAX_OUTBOUND_CONNECTIONS can default to higher then max_connections. Everything will still perform correctly but logging wouldn't be as clean as it should be.

If max_connections is 5 and MAX_OUTBOUND_CONNECTIONS default to 8 the log would show
Surpassed max inbound connections with maxconnections:5 minus max_outbound:8

Reply:
Yep. I missed them both. Thanks.

Reply:
Wait. I think you reviewed this right before I force pushed?

Reply:
Ok. I simplified the logging so it would not be misleading.